### PR TITLE
feat: Video Listing Page

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
- NavBar
- Aside
- VideoCard

NOTE:- The commented code will be used later on , so kindly bear with it

Screenshot (136)

### CODE REVIEW

**Review by vinc3nati — APPROVED** [review]
Overall looks good.
Good variable names.
Clean and Modular code

**Review by mansipadgelwar — APPROVED** [review]
Good to merge.

**Review by anuj-kumary — APPROVED** [review]

File: src/components/Aside/Aside.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import "./Aside.css"
+import { NavLink } from "react-router-dom"
+
+export const Aside=()=>{
+    return (
+        <aside className="sideBar flex-col"
+        // {sideBar? "showSide flex-col": "sideBar flex-col"}
```
Comment (ramalingaa) [inline src/components/Aside/Aside.jsx:7]: If not used remove commented code

File: src/components/Aside/Aside.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  background-color: var(--grey);
+}
+.aside-subhead {
+  font-size: 1.6rem;
+}
+
+/* @media screen and (max-width: 750px) {
+    .showSide {
+      position: absolute;
+      z-index: 10;
```
Comment (ramalingaa) [inline src/components/Aside/Aside.css:30]: Try not using random z-indices

File: src/components/NavBar/NavBar.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+.navbar {
+  padding: 0 3rem;
+  align-items: center;
+  justify-content: space-between;
+  height: 7rem;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  overflow: hidden;
+  border-bottom: 0.2rem solid #eee;
```
Comment (ramalingaa) [inline src/components/NavBar/NavBar.css:12]: Try using color variables

**Review by ramalingaa — APPROVED** [review]
Overall code looks good, try removing commented code before merging

**Pull request merged**